CONTRIBUTING.md

```diff
@@ -0,0 +1,199 @@
+
+While making your changes, commit often and write good, detailed commit messages.
+[This blog](https://chris.beams.io/posts/git-commit/) explains how to write a good Git commit
+message and why it matters.
+It is also perfectly fine to have a lot of commits - including ones that break code.
+A good rule of thumb is to push up to GitHub when you _do_ have passing tests then the continuous
+integration (CI) has a good chance of passing everything. 😸
+
+Please do not re-write history!
+That is, please do not use the [rebase](https://help.github.com/en/articles/about-git-rebase)
+command to edit previous commit messages, combine multiple commits into one, or delete or revert
+commits that are no longer necessary.
+
+### 4. Open a Pull Request
+
+We encourage you to open a pull request as early in your contributing process as possible.
+This allows everyone to see what is currently being worked on.
+It also provides you, the contributor, feedback in real-time from both the community and the
+continuous integration as you make commits (which will help prevent stuff from breaking).
+
+GitHub has a [nice introduction](https://guides.github.com/introduction/flow) to the pull request
+workflow, but please [get in touch](#get-in-touch) if you have any questions :balloon:.
+
+## First-timers' Corner
+
+If you're new to the project, we recommend starting with issues labeled
+as ["good first issue"](https://github.com/alan-turing-institute/autoemulate/issues?q=is:issue+is:open+label:%22good+first+issue%22).
+These are typically simpler tasks that offer a great starting point. Browse these here.
+
+There's also the
+label ["thoughts welcome"](https://github.com/alan-turing-institute/autoemulate/issues?q=is:issue+is:open+label:%22thoughts+welcome%22),
+which allows for you to contribute with discussion points in the issues, even if you don't want to
+or cannot contribute to the codebase.
+
+If you feel ready for it, you can also open a new issue. Before you open a new issue, please check
+if any of [our open issues](https://github.com/alan-turing-institute/autoemulate/issues) cover your idea
+already. If you open a new issue, please follow our basic guidelines laid out in our issue
+templates, which you should be able to see if
+you [open a new issue](https://github.com/alan-turing-institute/autoemulate/issues/new/choose).
+
+## Reporting Bugs
+
+Found a bug? Please open an issue here on GitHub to report it. We have a template for opening
+issues, so make sure you follow the correct format and ensure you include:
+
+- A clear title.
+- A detailed description of the bug.
+- Steps to reproduce it.
+- Expected versus actual behavior.
+
+## Recognising Contributions
+
+We value and recognize every contribution. All contributors will be acknowledged in the
+[contributors](https://github.com/alan-turing-institute/autoemulate/tree/main#contributors) section of the
+README.
+Notable contributions will also be highlighted in our fortnightly community meetings.
```

Comment:
maybe we take this out for now

Reply:
yep!

Reply:
I can write something about the sprint meetings here, and say that notable contributions will be highlighted in our demo sessions! :)